test/cpp/jit/test_interpreter.cpp

```diff
@@ -138,5 +142,38 @@ TEST(InterpreterTest, Basic_CUDA) {
   ASSERT_TRUE(exactlyEqual(outputs[0], hx));
   ASSERT_TRUE(exactlyEqual(outputs[1], cx));
 }
+
+TEST(InterpreterTest, runAsyncBasicTest) {
+  /*
+  TODO: there are some problem with C++ parsing script program involving
+  fork. Use the test module below for now.
+  The test module file is generated by following:
+    class DemoModule(torch.nn.Module):
+      def forward(self):
+        r1 = torch.jit.fork(torch.mm, torch.rand(100,100),torch.rand(100,100))
+        r2 = torch.jit.fork(torch.mm, torch.rand(100,100),torch.rand(100,100))
+        return r1.wait() + r2.wait()
+  demo = DemoModule()
+  torch.jit.save(torch.jit.script(demo), 'test_interpreter_async.pth')
+  */
+  auto model = load("caffe2/test/cpp/jit/test_interpreter_async.pth");
```

Comment:
this won't work in OSS CI - you need to change it to a relative path (or wait for the parser problem to be fixed)

examples:
https://github.com/pytorch/pytorch/blob/master/test/jit/test_save_load.py#L13
https://github.com/pytorch/pytorch/blob/master/test/quantization/test_backward_compatibility.py#L29

Reply:
I thought we are able to load the models by copying the generated code, so we are still not able to do so?

Reply:
forks seems to have problem with the parser.